content/sensu-go/6.9/operations/manage-secrets/secrets-providers.md

```diff
@@ -274,11 +355,157 @@ name: vault
 {{< /code >}}
 {{< /language-toggle >}}
 
-### Spec attributes
+### `CyberArkProvider` spec attributes
+
+client       | 
+-------------|------ 
+description  | Map that includes [CyberArkProvider client attributes][20].
+required     | true
+type         | Map of key-value pairs
+example      | {{< language-toggle >}}
+{{< code yml >}}
+client:
+  account: sensu.io
+  appliance_url: http://localhost:8480
+  login: host/Sensu/sensuBackend
+  api_key: CONJUR_API_KEY
+  timeout: 1s
+  tls:
+    ca_cert: "/etc/ssl/certs/conjur_ca_cert.pem"
+  ttl: 60s
+{{< /code >}}
+{{< code json >}}
+{
+  "client": {
+    "account": "sensu.io",
+    "appliance_url": "http://localhost:8480",
+    "login": "host/Sensu/sensuBackend",
+    "api_key": "CONJUR_API_KEY",
+    "timeout": "1s",
+    "tls": {
+      "ca_cert": "/etc/ssl/certs/conjur_ca_cert.pem"
+    },
+    "ttl": "60s"
+  }
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### `CyberArkProvider` client attributes
+
+account      | 
+-------------|------ 
+description  | Conjur account name.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+account: sensu.io
+{{< /code >}}
+{{< code json >}}
+{
+  "account": "sensu.io"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+appliance_url | 
+--------------|------ 
+description   | Conjur appliance URL.
+required      | true
+type          | String
+example       | {{< language-toggle >}}
+{{< code yml >}}
+appliance_url: http://localhost:8480
+{{< /code >}}
+{{< code json >}}
+{
+  "appliance_url": "http://localhost:8480"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+login        | 
+-------------|------ 
+description  | Conjur authentication login.
```

Comment:
Where do users get the value for the `login` attribute?

Reply:
The login and api_key are coupled. When we loaded the Conjur policy earlier it included a `Sensu` policy with a `sensuBackend` host. It then spit out an api key for sensuBackend.

host/Sensu/sensuBackend is as I understand it just the way resources get addressed in their product.